src/components/Group/Group.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
-import utils from '../utils';
+import utils from '../../utils';
 
-export const ButtonGroup = styled.div`
+export const Group = styled.div`
```

Comment:
Whats the reasoning with re-naming? It is this component going to be used for grouping other types of components?

Reply:
I was thinking in the future there could be field groups, or radio button groups, etc. So changing to Group made sense to me